remote_heartbeat:
  update_rate: 100
  latency_threshold_warn: 0.02    # s
  latency_threshold_error: 0.10   # s
